exporter/local/export.go

```diff
@@ -86,10 +88,30 @@ func (e *localExporterInstance) Export(ctx context.Context, inp exporter.Source)
 				if err != nil {
 					return err
 				}
+
+				idmap = mount.IdentityMapping()
+
 				defer lm.Unmount()
 			}
 
-			fs := fsutil.NewFS(src, nil)
+			walkOpt := &fsutil.WalkOpt{}
+
+			if idmap != nil {
+				walkOpt.Map = func(p string, st *fstypes.Stat) bool {
+					uid, gid, err := idmap.ToContainer(idtools.Identity{
```

Comment:
Shouldn't this be `ToHost` ? Since you are exporting, I'm assuming the uids of the files you're walking through need to be the ones on your host, not the ones in the container.

Reply:
converting from host IDs(remapped) to containerIDs (plain) on export.